core/examples/luigi-example-next/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "luigi-example-next",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
```

Comment:
Production build fails with Luigi on it: 

```
$ npm run build
...
Build error occurred
ReferenceError: Luigi is not defined
at Object.08Lt (/luigi/core/examples/luigi-example-next/build/server/pages/luigi-config.js:104:1)
```

This seems to be related to [routing rules in Next.js](https://nextjs.org/docs/routing/introduction). Since you put `luigi-config.js` inside the `pages` folder Next.js will attempt to build it on its own as a separate route (individually) and Luigi is not defined within that scope. 

I would suggest to move `luigi-config.js` outside of the `pages` folder and update the require path in `LuigiCoreApp.js` . This seems to solve that issue.

Reply:
Nice catch. I moved it into `public` folder. It works fine. Thanks